SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.7 BLOCK -->
```

Comment:
Where did this file come from?

Reply:
This was automatically added to the main branch by an MSFT bot in #10599. It's not present in the 2.8 branch

Reply:
> Where did this file come from?

Um I dunno, I think it's auto generated when I push into this branch